language/en/extimgaslink.php

```diff
@@ -23,5 +23,11 @@
 }
 
 $lang = array_merge($lang, array(
+	'EXTIMGASLINK_CONFIG'			=> 'Allowed origin of images used in posts',
```

Comment:
Das ist ziemlich lang - vielleicht besser:
> Permitted images

Das wäre die direkte Übersetzung. Alternativ:
> Permitted image origins

Der Rest steht ja dann in der Erklärung.